src/pywifes.py

```diff
@@ -1862,7 +1862,7 @@ def wifes_slitlet_mef_ns(inimg, outimg_obj, outimg_sky,
         # create fits hdu for object
         hdu_name = 'SCI%d' % (i+1)
         obj_hdu = pyfits.ImageHDU(obj_data, old_hdr, name=hdu_name)
-        obj_hdu.header.set('CCDSEC',  obj_dim_str)
+        obj_hdu.header.set('DETSEC',  obj_dim_str)
         obj_hdu.header.set('DATASEC', obj_dim_str)
         obj_hdu.header.set('TRIMSEC', obj_dim_str)
         # fix the exposure time!!
```

Comment:
Missed exptime_true = float(old_hdr['SEXP'])*float(old_hdr['NSUBEXPS']) just below? Not getting any errors? Do tests not currently hit `wifes_slitlet_mef_ns`?

Reply:
Good spotted! I just checked and in the current configuration, the tests do not hit `wifes_slitlet_mef_ns` as the argument `ns` in `slitlet_mef` is set to False but I guess there could be cases where this argument could be set to True.
In any case, I changed the header names and also spotted another one further down also in pywifes.py.